unittest/gunit/CMakeLists.txt

```diff
@@ -317,6 +322,13 @@ IF (MERGE_UNITTESTS)
   SET_SOURCE_FILES_PROPERTIES(MERGE_SMALL_TESTS MERGE_LARGE_TESTS
     PROPERTIES GENERATED 1)
 
+  # Suppress -Wformat-overflow= for my_safe_snprintf(" %s ", nullptr)
+  IF(HAVE_FORMAT_OVERFLOW)
+    ADD_COMPILE_FLAGS(
+      ${MERGE_LARGE_TESTS}
+      COMPILE_FLAGS "-Wno-format-overflow")
```

Comment:
With `set_source_files_properties(segfault-t.cc PROPERTIES COMPILE_FLAGS -Wno-format-overflow)`, you don't have to duplicate this.

Reply:
It's backported from 5.7 so we may keep it just to minimize a diff.